Obsidian/Utilities/Config.cs

```diff
@@ -37,6 +37,15 @@ public class Config : IConfig
 
     public string[] DownloadPlugins { get; set; } = Array.Empty<string>();
 
+    public bool EnableRcon { get; set; } = false;
+
+    public string RconPassword { get; set; } = string.Empty;
+
+    public ushort RconPort { get; set; } = 25575;
+
+    public bool BroadcastRconToOps { get; set; } = true;
+    public bool RequireRconEncryption { get; set; } = false;
+
```

Comment:
Would consider moving these properties into their own class so the resulting json would look like

```json
{
  "rcon": {
    "enabled": true,
    "password": "xyz",
    "port": 6969,
    "broadcastCommandsToOperators": false,
    "requireEncryption": false 
  }
}
```

Reply:
Possible